convex/migrations.ts

```
@@ -1182,3 +1182,86 @@ export const rollbackMigration = internalMutation({
+    const allUserSettings = await ctx.db.query("userSettings").collect();
+
+    const results = {
+      total: allUserSettings.length,
+      cleaned: 0,
+      errors: 0,
+    };
+
+    for (const userSetting of allUserSettings) {
+      try {
+        // Check if the document has tour-related fields
+        const doc = userSetting as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)